### DISCUSSION TITLE
YT Link Videos Downloading at Low Quality | yt-dlp

### DISCUSSION THREAD

**Discussion by stuartle** [Serene-Arc/bulk-downloader-for-reddit#961] [Q&A]
All data formats download at full correct resolution, but am having trouble specifically with youtube links being downloaded at low  resolution.

I understand that bdfr calls yt-dlp to handle download of youtube links, and I've tried using yt-dlp itself to download the same links, which will correctly download HD video. 

Any idea of any configuration that may affect this?

**Comment by Serene-Arc** [comment]
We use the 'best'' setting for `yt-dlp` in the code for the downloader. Perhaps try using that and see what downloads with yt-dlp normally. If it's the low resolution, then open a bug and I'll have a look.
